smif/controller.py

```diff
@@ -206,8 +206,12 @@ def load_timesteps(self, file_path):
         list
             A list of timesteps
         """
-        logger.info("Loading timesteps from {}".format(file_path))
-        self.sos_model._timesteps = ConfigParser(file_path).data
+        logger.info("Loading timesteps from %s", file_path)
```

Comment:
Mainly to keep the linter quiet. I understand it's a minor performance thing: the `{}` style means the string is always evaluated, whatever the logging level, whereas the logger can decide whether or not to do the string interpolation if you pass it with the `%s` style and arguments.